### PR TITLE
Add address to payments and orders

### PR DESCRIPTION
**Pull request by dlahyani**
* Add billing_address to stg_payments (taken from raw_customers)
* Add shipping_address to stg_orders (taken from raw_customers)
* Change shipping_address in orders to use shipping_address from stg_orders.

### CODE REVIEW

**Comment by galunto** [comment]
!refresh

**Comment by galunto** [comment]
!refresh

File: models/staging/stg_orders.sql [hunk 1]
```diff
@@ -8,15 +8,25 @@ with source as (
 
 ),
 
+customer_address as (
+    select
+        id,
+        address,
+    from  {{ ref('raw_customers') }
```
Comment (merlindso) [inline models/staging/stg_orders.sql]:
```suggestion
    from  {{ ref('raw_customers') }}
```

**Comment by galunto** [comment]
!refresh

File: models/staging/stg_payments.sql [hunk 2]
```diff
@@ -4,22 +4,36 @@ with source as (
...
+    from {{ ref('raw_payments') }}
+    join {{ ref('raw_orders') }}
+    on {{ ref('raw_payments') }}.order_id = {{ ref('raw_orders') }}.id
 ),
 
+customer_address as (
+    select
+        id,
+        address,
+    from  {{ ref('raw_customers') }
```
Comment (merlindso) [inline models/staging/stg_payments.sql]:
```suggestion
    from  {{ ref('raw_customers') }}
```

**Comment by galunto** [comment]
Comment

**Pull request closed without merging**